packages/react-dom-bindings/src/shared/ReactFlightClientConfigDOM.js

```diff
@@ -10,4 +10,38 @@
 // This client file is in the shared folder because it applies to both SSR and browser contexts.
 // It is the configuraiton of the FlightClient behavior which can run in either environment.
 
-// In a future update this is where we will implement `dispatchDirective` such as for Float methods
+import type {Directive} from '../server/ReactFlightServerConfigDOM';
+
+import ReactDOMSharedInternals from 'shared/ReactDOMSharedInternals';
+const ReactDOMCurrentDispatcher = ReactDOMSharedInternals.Dispatcher;
+
+export function dispatchDirective(directive: Directive): void {
```

Comment:
There's a lot of things named "dispatch" and "dispatchDirective" which is a little unclear when it's emitting vs executing something.

`executeDirective` maybe.

Reply:
we settled on Hints and emitHint (server) and dispatch (client). The dispatch aligns with the dispatcher making something happen with this hint